.github/workflows/ci.yml

```diff
@@ -22,4 +22,5 @@ jobs:
       - name: run tests
         run: |
           ./bazel build //...
-          ./bazel test --test_output=errors //... --test_env=COURSIER_CACHE="${{ github.workspace }}/.coursier-cache"
\ No newline at end of file
+          ./bazel test --test_output=errors //... --test_env=COURSIER_CACHE="${{ github.workspace }}/.coursier-cache"
+          ./bazel run //:parse -- generate -r `pwd` -s 3rdparty/workspace.bzl -d dependencies.yaml -t 3rdparty/target_file.bzl -p pom.xml --check-only
```

Comment:
Is this `--check-only` feature documented in README?

Reply:
it is not currently. Only the cli help it looks like.